scripts/release/setup_delivery_config.py

```diff
@@ -35,12 +35,12 @@ def setup_delivery(cfg_dir: Path, mode: ProcessingMode):
         " --dem ./dynamic_ancillary_files/dem.tif"
         # Geometry files/static layers
         " --geometry-files ./dynamic_ancillary_files/static_layers/*"
-        " --mask-file ./dynamic_ancillary_files/watermask.tif"
+        " --mask-file ./dynamic_ancillary_files/water_mask.tif"
         #
         # Unwrapping stuff
         " --unwrap-method snaphu --ntiles 5 5 --downsample 5 5"
         # Worker stuff
-        " --threads-per-worker 16 --n-parallel-bursts 2 --n-parallel-unwrap 2 "
+        " --threads-per-worker 16 --n-parallel-bursts 4 --n-parallel-unwrap 4 "
```

Comment:
just noting here that since we haven't kept parity with all the configuration options, `n_parallel_tiles` isn't exposed so is 1

Reply:
Ah ok so doesn't matter